### DISCUSSION TITLE
hyphenated path param ids break regex creation

### DISCUSSION THREAD

**Discussion by mordam** [cdimascio/express-openapi-validator#899] [General]
Have been wiring in eov into a service. It worked great until i configured a path like

/someurl/{parameter-id}}

this resulted in "NotFound" because the regex that gets created to do the matching ended up with a literal '-id' in it and would not match

/someurl/aParamter

Either a bug or a documentation update. Thoughts?

**Comment by cdimascio** [comment]
Hmm. Seems like a bug in the regex pattern.

**Reply by mordam** [reply]
Yes absolutely. 

I think such a var name should be supported, so would suggest opening a bug.
